File: Model/view_customer.py
#!c:\python\python
from lib import DBConnection as c
import cgi, cgitb,json,os
print("Content-type: text/html\n\n")
cgitb.enable()
form = cgi.FieldStorage()
action=form.getvalue("action")
if action == "getitem":
    sql = "select  name,email,place,phone_no from customer b"
    res = c.db.getJSON(sql)
    print(res)

File: Controller/view_customer.js
$(function () {
    GetCustomer()
})
function GetCustomer() {
    
    data = { "action": "getitem" }
    console.log(data)
    $.post("Model/view_customer.py", data, function (res) {
        res = JSON.parse(res)
        console.log(res)
        if (res != null) {
            $("#tb").show()
            $.each(res, function (k, v) {
                s = '<tr><td>' + v.name + '</td>' +
                    '<td>' + v.email + '</td>' +
                    '<td>' + v.place + '</td>' +
                    '<td>' + v.phone_no + '</td>' 
                    
                $("#stds").append(s)
            })
        } else {
            
         
        }
    })
}